src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, useMemo, useState } from "react";
```

Comment:
This file looks a lot like a copy-paste of `AdminVenueView.tsx `. Is there a reason for that?

Reply:
Both screens look identical. The only difference is the object that's being mapped. I'd like to create a follow up PR which will create a reusable screen with tabs, and you can just pass the object. That way it can be reused.